ml-deep-dive/svm.md

```diff
@@ -0,0 +1,96 @@
+# Support Vector Machines
+Support vector machines (SVMs) are another subset of supervised learning methods that can be used for either classification or regression tasks. To find out what makes Support Vector Machines unique, and get a more expansive overview of SVMs, we would encourage students to follow the tutorial linked below:  
+[Support Vector Machines Tutorial – Learn to implement SVM in Python](https://data-flair.training/blogs/svm-support-vector-machine-tutorial/)
+> This tutorial will guide you through the following understanding:
+> * What are SVMs?
+> * A visual introduction to how they work
+> * Implementation in Python
+> * Pros and Cons of this algorithm
+> * Model Parameter Tuning
+> * Applications of SVM
+
+## What makes SVMs such an important algorithm?
+1. SVMs are highly effective in high-dimensional spaces, making them one of the most frequently used solutions for dimensional data.
+2. SVMs tend to generally find an optimal solution due to underlying convex optimization.
+3. SVMs can deal with linearity or non-linearity in the data, by choice of appropriate kernels
+4. SVMs make use of the the **Kernel Trick** to handle non-linearly separable data
+### Kernel Trick
+> A kernel is simply a transformation of a two-dimensional plane to a higher dimensional space.
+
+Sometimes separability cannot be achieved in a linear space. By application of a kernel trick, non-linear data is projected in to a higher dimensional space, where separation is much easier.
+
+![kernel-trick](./svm_kernel_trick.jpg)
```

Comment:
I would suggest making a new folder and putting it in their :)

Reply:
- [ ] Move images to dedicated folder

Reply:
lgtm (once image is in asset folder)